src/Message/RefundRequest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace MyOnlineStore\Omnipay\KlarnaCheckout\Message;
+
+use Klarna\Rest\OrderManagement\Order;
+
+final class RefundRequest extends AbstractRequest
+{
+    use ItemDataTrait;
+
+    /**
+     * @inheritDoc
+     */
+    public function getData()
+    {
+        $this->validate('transactionReference', 'amount');
+
+        return [
+            'refunded_amount' => $this->getAmountInteger(),
+            'order_lines' => $this->getItemData($this->getItems()),
```

Comment:
This `getItemData` call should probably be conditional too, like how it is being used in the `CaptureRequest` class.